client/src/app/index.js

```diff
@@ -1,10 +1,12 @@
 // npm packages
 import React from 'react';
 
+import Header from '../components/header';
 import Footer from '../components/footer';
 
 export default ({children}) => (
   <div className="container">
+    <Header />
```

Comment:
But this way header will be visible when user is not authenticated. And we don't want that because it makes it confusing for a user.